contracts/Orchestrator.sol

```diff
@@ -0,0 +1,165 @@
+pragma solidity 0.4.24;
+
+import "openzeppelin-eth/contracts/ownership/Ownable.sol";
+
+import "./UFragmentsPolicy.sol";
+
+
+/**
+ * @title Orchestrator
+ * @notice The orchestrator is the main entry point for rebase operations. It coordinates the policy
+ * actions with external consumers.
+ */
+contract Orchestrator is Ownable {
+
+    struct Transaction {
+        address destination;
+        uint value;
```

Comment:
How about hardcoding value to 0?

Reply:
I guess it does get set to 0 by default. 
No harm in being explicit.
We can do it in the constructor

Reply:
I think the value can only be set on instantiation

Reply:
It's set once when the transaction is created and added to the list. 
Do you mean just set the initial value?
Or make it so it's impossible to add a value?

Reply:
Isn't value an amount of Ether to send? I meant completely disable the option to send Ether on these calls, as I don't see a use case for it. Hardcode value to 0 in externalCall

Reply:
Yeah, you're right in the sense that the first test ensures the contract can't even receive ETH to begin with.

I can either make it so it can receive ETH, or make the value not configurable to begin with. I think I'll go with the second one.

Do you agree @nithinkrishna ?

Reply:
2nd one makes sense 👍